src/tracer.js

```diff
@@ -254,23 +257,26 @@ export default class Tracer {
     }
 
     /**
-     *  Makes a concrete sampling decision for the ctx span context based on information
-     *  available to it's child span.
-     *
-     *  The ctx context's deferred mask is unset after this decision is made
+     * Reapplies sampling if the parent span isn't sampled
      *
+     * @param config the upsampling config
```

Comment:
does the api need to have config passed in? given config is available to `this`? is this just for tests?